src/osd/osd_types.cc

```diff
@@ -100,6 +100,12 @@ const char * ceph_osd_op_flag_name(unsigned flag)
     case CEPH_OSD_OP_FLAG_FADVISE_NOCACHE:
       name = "fadvise_nocache";
       break;
+    case CEPH_OSD_OP_FLAG_WITH_REFERENCE:
+      name = "reference_couting";
```

Comment:
Don't want to be nit-picking, but I'd prefer `with_reference `to keep pace with others...

Reply:
Really never mind, your suggestions are good and reasonable, indeed I should appreciate your patience! It's me not doing well, thanks :100: